### PR TITLE
Pad fields in RevisionHeader with spaces instead of tabs

### PR DESCRIPTION
**Pull request by Carbenium**
- Mono didn't implement SelectionTabs for RichTextBox, so tabs aren't an option
- Monospaced font has to be used for the text, otherwise the padding won't work

### CODE REVIEW

**Comment by Carbenium** [comment]
rebased

File: GitUI/CommitInfo/CommitInfo.cs [hunk 1]
```diff
@@ -33,6 +33,7 @@ public CommitInfo()
             {
                 _sortedRefs = null;
             };
+            _RevisionHeader.Font = new System.Drawing.Font(System.Drawing.FontFamily.GenericMonospace, AppSettings.Font.Size);
```
Comment (jbialobr) [inline GitUI/CommitInfo/CommitInfo.cs]: If the user has already chosen a monospaced font, it should not be changed to some other monospaced one.

**Comment by jbialobr** [comment]
Or even better: create an RTF table to layout the commit header data.

**Comment by Carbenium** [comment]
I'm not sure if RTF is fully implemented in mono. I'd rather keep it simple.

**Comment by RussKie** [comment]
@Carbenium sorry it took this long.

**Pull request closed without merging**